integrations/atlassian/confluence/event.go

```diff
@@ -126,7 +127,10 @@ func extractEntityType(l *zap.Logger, atlassianEvent map[string]any, category st
 		}
 	}
 
-	// Some "relation_*" events look a little different.
+	// Some "attachment_*" and "relation_*" events look a little different.
+	if _, ok := atlassianEvent["attachments"]; ok {
+		return fmt.Sprintf("attachment_%s", category), true
+	}
 	if _, ok := atlassianEvent["relationData"]; ok {
 		return fmt.Sprintf("relation_%s", category), true
 	}
```

Comment:
So the event can only have either `attachments` or `relationData`?

Reply:
Explained offline.